zulip.md

```diff
@@ -0,0 +1,66 @@
+# Zulip
+
+##  Getting started on Zulip:
+
+### If you’re new here, here’s what you can do to get started with this community:
+
+1.  :heavy_check_mark: Introduce yourself on [#newcomers](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) stream in the [#newcomers > Introductions](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers/topic/Introductions) topic and tell us your motivations for joining so we can get to know you and help you on your journey.
+
+2.   :calendar: Follow our Open Sessions in Calendar Events topic on [#announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements) or through our AnitaB.org Open Source calendar ([PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=GMT) || [WAT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Africa/Lagos) || [IST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Asia/Colombo) || [AEDT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Australia/Sydney)). These are voice calls you can participate in and get to know a bit more our community and projects.
+
+##   Learning Resources :eyeglasses:
+### To keep this community safe for you and others.
```

Comment:
Is `safe` correct term to use?

Reply:
Hi @jhalak27 , I have fixed the typos,  Let me know if I should use the term safe or not.

Reply:
Maybe 'helpful'. Not sure.
@isabelcosta @foongminwong what you suggest?

Reply:
Suggestion: `These are voice call sessions you can participate to learn more about our community and projects.`

Reply:
> Maybe 'helpful'. Not sure.
> @isabelcosta @foongminwong what you suggest?

Hmm I'm also not sure what word can be used, or we could remove line 13?